src/plugins/orangehrmDashboardPlugin/Dao/ChartDao.php

```diff
@@ -130,6 +130,7 @@ public function getEmployeeDistributionByLocation(): array
         $q->andWhere($q->expr()->isNull('employee.purgedAt'));
         $q->addGroupBy('location.id');
         $q->addOrderBy('COUNT(employee.empNumber)', 'DESC');
```

Comment:
```suggestion
        $q->addOrderBy('COUNT(employee.empNumber)', \OrangeHRM\ORM\ListSorter::DESCENDING);
```

Reply:
Add to https://github.com/orangehrm/orangehrm/pull/1440